### ISSUE TITLE
Trackable log  page not loading

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#6113]
- Search for TB private code from main screen search bar
- TB page opens normally
- Start logging

It seems cgeo cannot load the needed data: The progress spinner keeps spinning continously...waited for 1 minute.

No debug log available at the moment from my side

Loading the TB log page in browser works normal

Current nightly

**Comment by Lineflyer** [comment]
I took a deeper look into this:

I do see in the log that the page is requested ad retrieved, but still the progress spinner does not stop and also after waiting for 1 minute nothing else happend according to the log.

So I just tried sending the TB-log which resulted in a crash.
Attached the log for the whole sequence (notice the wait time acc. to the time stamps):

The secret code is not really secret...if someone sees it here, he is welcome to discover as it is my owned trackable (error looks the same for not owned TBs).

```
22:19:13.497 Debug cgeo 15516  [OkHttp] GET https://www.geocaching.com/track/details.aspx?tracker=5A2NVA
22:19:13.716 Debug cgeo 15516  [OkHttp] 200 (251 ms) GET https://www.geocaching.com/track/details.aspx?tracker=5A2NVA (http/1.1)
22:19:16.664 Debug cgeo 15516  [OkHttp] GET https://www.geocaching.com/track/details.aspx?tracker=TB59M9P
22:19:16.992 Debug cgeo 15516  [OkHttp] 200 (310 ms) GET https://www.geocaching.com/track/details.aspx?tracker=TB59M9P (http/1.1)
22:21:54.786 Error cgeo 15516  [main] UncaughtException
22:21:54.786 Error cgeo 15516  java.lang.NullPointerException: Attempt to invoke virtual method 'void cgeo.geocaching.models.Trackable.setTrackingcode(java.lang.String)' on a null object reference
22:21:54.786 Error cgeo 15516  	at cgeo.geocaching.log.LogTrackableActivity.sendLog(LogTrackableActivity.java:619)
22:21:54.786 Error cgeo 15516  	at cgeo.geocaching.log.LogTrackableActivity.onOptionsItemSelected(LogTrackableActivity.java:582)
22:21:54.786 Error cgeo 15516  	at android.app.Activity.onMenuItemSelected(Activity.java:3024)
22:21:54.786 Error cgeo 15516  	at android.support.v4.app.FragmentActivity.onMenuItemSelected(FragmentActivity.java:325)
22:21:54.786 Error cgeo 15516  	at android.support.v7.app.ActionBarActivity.superOnMenuItemSelected(ActionBarActivity.java:245)
22:21:54.786 Error cgeo 15516  	at android.support.v7.app.ActionBarActivityDelegateICS.onMenuItemSelected(ActionBarActivityDelegateICS.java:164)
22:21:54.786 Error cgeo 15516  	at android.support.v7.app.ActionBarActivity.onMenuItemSelected(ActionBarActivity.java:131)
22:21:54.786 Error cgeo 15516  	at android.support.v7.app.ActionBarActivityDelegateICS$WindowCallbackWrapper.onMenuItemSelected(ActionBarActivityDelegateICS.java:308)
22:21:54.786 Error cgeo 15516  	at com.android.internal.policy.impl.PhoneWindow.onMenuItemSelected(PhoneWindow.java:1208)
22:21:54.786 Error cgeo 15516  	at com.android.internal.view.menu.MenuBuilder.dispatchMenuItemSelected(MenuBuilder.java:761)
22:21:54.786 Error cgeo 15516  	at com.android.internal.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:155)
22:21:54.786 Error cgeo 15516  	at com.android.internal.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:904)
22:21:54.786 Error cgeo 15516  	at com.android.internal.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:894)
22:21:54.786 Error cgeo 15516  	at android.widget.ActionMenuView.invokeItem(ActionMenuView.java:611)
22:21:54.786 Error cgeo 15516  	at com.android.internal.view.menu.ActionMenuItemView.onClick(ActionMenuItemView.java:180)
22:21:54.786 Error cgeo 15516  	at android.view.View.performClick(View.java:5197)
22:21:54.786 Error cgeo 15516  	at android.view.View$PerformClick.run(View.java:20926)
22:21:54.786 Error cgeo 15516  	at android.os.Handler.handleCallback(Handler.java:739)
22:21:54.786 Error cgeo 15516  	at android.os.Handler.dispatchMessage(Handler.java:95)
22:21:54.786 Error cgeo 15516  	at android.os.Looper.loop(Looper.java:145)
22:21:54.786 Error cgeo 15516  	at android.app.ActivityThread.main(ActivityThread.java:5951)
22:21:54.786 Error cgeo 15516  	at java.lang.reflect.Method.invoke(Native Method)
22:21:54.786 Error cgeo 15516  	at java.lang.reflect.Method.invoke(Method.java:372)
22:21:54.786 Error cgeo 15516  	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1400)
22:21:54.786 Error cgeo 15516  	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1195)
```

**Comment by Lineflyer** [comment]
Add on:
Nightly from 2016.10.23 is working OK.

**Comment by Lineflyer** [comment]
Anyone an idea what is wrong here?

**Comment by samueltardieu** [comment]
I might.

**Comment by Lineflyer** [comment]
Confirmed fixed with 2016.11.15-NB1 including the mentioned change.
